samples/GetUserProfile.py

```diff
@@ -58,7 +38,7 @@
 if len(sys.argv) > 4:
     userid = sys.argv[4]
 else:
-    userid = get_input("UserID: ")
```

Comment:
I was on the fence about this. I wanted to make it clear what was sdk code and what was sample code, which is why I liked samples_common.blah_blah_blah though I can see an argument for clarity too.